xml/vt_scenarios.xml

```diff
@@ -0,0 +1,198 @@
+   is shared between VM.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Secure Boot can be used for VM
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     KSM should be avoided
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     CPU can be assigned to a VM, and HT should be disabled to avoid security issue
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     VM should not share network / storage / network / Hardware
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Migration capabilities could be impacted in case of specific usage like PCI passthrough or NUMA
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Full Virtualization is generally less efficient if no PV/virtio drivers are used
+    </para>
+   </listitem>
+  </itemizedlist>
+<para>
+    AMD provides some specific features regarding Virtualization security.
+    <!-- link to the SEV doc? -->
+   </para>
+  <itemizedlist>
+   
+   <listitem>
+    <para>
+     AMD Secure Memory Encryption (SME):
+     Uses a single key to encrypt system memory. The key is generated by the AMD Secure Processor at boot. SME requires enablement in the system BIOS or operating system. When enabled in the BIOS, memory encryption is transparent and can be run with any operating system.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     AMD Secure Encrypted Virtualization (SEV):
+     Uses one key per virtual machine to isolate guests and the hypervisor from one another. The keys are managed by the AMD Secure Processor. SEV requires enablement in the guest operating system and hypervisor. The guest changes allow the VM to indicate which pages in memory should be encrypted. The hypervisor changes use hardware virtualization instructions and communication with the AMD Secure processor to manage the appropriate keys in the memory controller.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     AMD Secure Encrypted Virtualization-Encrypted State (SEV-ES):
+     Encrypts all CPU register contents when a VM stops running. This prevents the leakage of information in CPU registers to components like the hypervisor, and can even detect malicious modifications to a CPU register state.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <!-- MIGRATION IS A VT FEATURE NOT A SCENARIO
+      <sect1>
+      <title>Migration</title>
+      <para>
+      Migration is a process to move a running virtual machine to another
+      physical machine. Live migration is an extended feature that allows
+      this move without disconnection of the client or the application.
+      </para>
+      </sect1>
+ -->
+ <sect1>
+  <title>Disaster Recovery</title>
+  <para>
+   <xref linkend="gloss-vt-virtualized"/> guests are less dependent on
+   physical hardware. The host server supports making snapshots
+   of the virtual machines, enabling restoration to a known good state,
+   or to any desired earlier state.
```

Comment:
```suggestion
   The hypervisor can make snapshots of VMs, enabling restoration
   to a known good state, or to any desired earlier state. Since   
   <xref linkend="gloss-vt-virtualized"/> OSes are less dependent on
   hardware configuration than those running directly on bare metal,
   these snapshots can be restored onto different server hardware so
   long as it is running the same hypervisor.
```